src/apps/authentication/errors.py

```diff
@@ -27,8 +27,19 @@ class PermissionsError(AccessDeniedError):
 
 
 class EmailDoesNotExist(AccessDeniedError):
-    message = _("Incorrect Email.")
+    message = _("That email is not associated with a MindLogger account.")
 
 
 class PasswordMismatch(AccessDeniedError):
-    message = _("Incorrect Password.")
+    def __init__(self, email=None):
+        self.email = email
+        super().__init__(email=email)
+
+    @property
+    def message(self):
+        message = _("Incorrect Password.")
+        if self.email:
+            message = (
+                f"Incorrect password for {self.email} if that user exist."
```

Comment:
According to pentest report the system should return the same error for not existing email or incorrect password for preventing user enumeration vulnerability